holoviews/plotting/bokeh/raster.py

```diff
@@ -21,21 +21,13 @@ def __init__(self, *args, **kwargs):
         if self.hmap.type == Raster:
             self.invert_yaxis = not self.invert_yaxis
 
-
-    def _glyph_properties(self, plot, element, source, ranges):
-        properties = super(RasterPlot, self)._glyph_properties(plot, element,
-                                                               source, ranges)
-        properties = {k: v for k, v in properties.items()}
+    def get_data(self, element, ranges, style):
+        mapping = dict(image='image', x='x', y='y', dw='dw', dh='dh')
         val_dim = [d for d in element.vdims][0]
```

Comment:
Not part of the PR but ``val_dim = [d for d in element.vdims][0]`` looks off to me. Why not ``val_dim = element.vdims[0]``?

Reply:
True not sure what the point of that is. Assume it used to grab the dimension name and then changed.